stonesoup/simulator/tests/conftest.py

```diff
@@ -30,3 +36,52 @@ def function(state, noise):
             matrix = np.array([[1, 0, 0, 0], [0, 0, 1, 0]])
             return matrix @ state.state_vector
     return TestMeasurementModel()
+
+
+@pytest.fixture()
+def sensor_model1():
+    class TestSensor():
+
+        @staticmethod
+        def measure(ground_truth):
+            return Detection(ground_truth.state_vector[(0, 2), :],
+                             timestamp=ground_truth.timestamp)
+    return TestSensor()
+
+
+@pytest.fixture()
+def sensor_model2():
+    class TestSensor:
+
+        @staticmethod
+        def measure(ground_truth):
```

Comment:
The problem with this is that you are not actually testing your simulator against the platform/sensor classes that people will use. 

Implementing your own `measure()` function is fine, but I would subclass Sensor
```suggestion
class TestSensor(Sensor):
   def measure(self, ground_truth):  # Note not static as in Sensor
````

Suggestion for Platform below